docs/BR.md

```diff
@@ -1888,7 +1888,10 @@ h. __Certificate Field:__ `subject:countryName` (OID: 2.5.4.6)
 
 i. __Certificate Field:__ `subject:organizationalUnitName` (OID: 2.5.4.11)
    __Required/Optional:__ __Optional__.  
-   __Contents__: The CA SHALL implement a process that prevents an OU attribute from including a name, DBA, tradename, trademark, address, location, or other text that refers to a specific natural person or Legal Entity unless the CA has verified this information in accordance with Section 3.2 and the Certificate also contains `subject:organizationName`, `subject:givenName`, `subject:surname`, `subject:localityName`, and `subject:countryName` attributes, also verified in accordance with Section 3.2.2.1.
+   __Contents__: The CA SHALL implement a process that prevents an OU attribute from including a name, DBA, tradename, trademark, address, location, or other text that refers to a specific natural person or Legal Entity unless the CA has verified this information in accordance with Section 3.2 and the Certificate also contains the following attributes, verified in accordance with section 3.2.2.1:
+	1.	subject:organizationName, OR;
+	2.	subject:givenName AND subject:surname
+
```

Comment:
I would put it more in line with how `subject:postalCode` is written:

```suggestion
   __Required/Optional:__  
   __Optional__ if the `subject:organizationName` field or both of the `subject:givenName` and `subject:surname` fields are present.  
   __Prohibited__ if both the `subject:organizationName` field and either of the `subject:givenName` and `subject:surname` fields are absent.  
   __Contents:__ If present, the `subject:organizationalUnitName` field MUST contain the Subject's organizational unit name as verified under Section 3.2.2.1.
```

This _does_ prohibit the 'Domain Control Validated' class of OU fields, but metadata-only subject fields are already prohibited, and it is my _opinion_ that any data other than what is validated through 3.2.2 is metadata relative to the subject; this includes the methods from 3.2.2 used to retrieve or validate subject data.